atom/browser/api/atom_api_dialog.cc

```diff
@@ -92,6 +93,9 @@ void ShowSaveDialog(const std::string& title,
                     const std::string& button_label,
                     const base::FilePath& default_path,
                     const file_dialog::Filters& filters,
+                    const std::string& message,
+                    const std::string& name_field_label,
+                    const bool& shows_tag_field,
```

Comment:
For primitives like `bool` and `int`, we don't use references or `const`, so this should just be `bool show_tags_field`.